pkg/rorresources/rortypes/resourcedef_backupJob.go

```diff
@@ -0,0 +1,50 @@
+package rortypes
+
+import "time"
+
+type ResourceBackupJob struct {
+	SourceName            string                `json:"sourceName"` // Defines the name of the system the run originates from
```

Comment:
Should the name and ids be placed in spec instead? and ResourceBackupRuns moved under status?
ex:

```yaml
spec:
    name
    sourceId
    ...
status:
    Runs
```

Reply:
After talking with Roger this needs to reworked. Back to the drawing board.

Reply:
@rogerwesterbo I've updated the model to reflect on more what we talked about, and changed some structs/fields after having given them another thought. If you can take another look when you have the time that'd be great.